packages/babel-plugin-proposal-destructuring-private/test/fixtures/ordering/.array-pattern-side-effect-iterable/exec.js

```diff
@@ -0,0 +1,29 @@
+var log = [];
```

Comment:
This test is disabled because currently the plugin does not support it: in `transformPrivateKeyDestructuring` the sub patterns of the array pattern as LHS is evaluated after the iterable at RHS is consumed.